.ci/static-checks.sh

```diff
@@ -647,6 +647,9 @@ static_check_docs()
 	# since it displayed by default when visiting the repo.
 	exclude_doc_regexs+=(^README\.md$)
 
+	# Exclude documentation files in vendor directory
+	exclude_doc_regexs+=(^vendor/.*\.md$)
+
```

Comment:
Hmm, see, I think/expect the vendor files to already be stripped out by the `skip_paths()` call earlier, which uses the ['paths_to_skip`](https://github.com/kata-containers/tests/blob/master/.ci/static-checks.sh#L57-L60) list to prune things out...

```bash
paths_to_skip=(
	"${ignore_clh_generated_code}"
	"vendor"
)
```

I thought the `exclude_doc_regexs` was to remove *our* files that we know we do not check (like license files etc.). Sooo, just want to check that this fix is not papering over a crack in the wrong place?

/cc @jodh-intel

Reply:
@grahamwhaley you're right, the issue was at another place. Fixed the `skip_paths` function. ptal.